CHANGELOG.md

```diff
@@ -8,17 +8,17 @@ This file is used to list changes made in each version of the haproxy cookbook.
 - Fix cookbook default value in `config_global`
 - Clean up unit and integration test content regular expressions
 - Adds ius repo for CentOS and Redhat package installations (resolves #348)
+- Systemd unit configured and working
```

Comment:
You probably want a n action before this, adds/fixes etc

Reply:
i may just drop this line in favor of line 14 `- Remove `poise_service` dependency in favor of systemd_unit`